evap/staff/importers/enrollment.py

```diff
@@ -555,6 +588,40 @@ def finalize(self) -> None:
             )
 
 
+class UserDegreeMismatchChecker(Checker, RowCheckerMixin):
+    """Assert that a users degree is consistent between rows"""
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+        self.degree_by_email: Dict[str, Degree] = {}
+        self.tracker: FirstLocationAndCountTracker = FirstLocationAndCountTracker()
```

Comment:
Not sure what style we want to prefer, but I would lean towards omitting type annotations that can be inferred:
```suggestion
        self.tracker = FirstLocationAndCountTracker()
```
How do you feel about it?